finance_dl/amazon.py

```diff
@@ -169,7 +169,8 @@ def invoice_finder():
 
                 order_ids = set()
                 for invoice_link in invoices:
-                    if "nvoice" not in invoice_link.text:
+                    # Amazon Fresh, and regular orders respectively
+                    if invoice_link.text not in ("View order", "View invoice"):
```

Comment:
Unfortunately I don't recall why I found this change necessary.

Reply:
I think it's clear in the new approach why it's required :)